crates/fluvio/src/config/cluster.rs

```diff
@@ -71,3 +143,255 @@ impl TryFrom<FluvioConfig> for fluvio_socket::ClientConfig {
         ))
     }
 }
+
+#[cfg(test)]
+mod test_metadata {
+    use serde::{Deserialize, Serialize};
+    use crate::config::Config;
+
+    #[test]
+    fn test_get_metadata_path() {
+        let toml = r#"version = "2"
+[profile.local]
+cluster = "name"
+
+[cluster.name]
```

Comment:
This examples is not right.  All cluster should have specific name. So it would be something like
```
[cluster.local.metadata.custom]
```

Reply:
but that's exactly what it's doing, right?

the name of the cluster in this example is "name": `[cluster.name.metadata.custom]`

Reply:
that's wasn't clear.  typically you don't have `name` as name 😉

Reply:
I renamed the cluster name to "local" to make it more clear